### DISCUSSION TITLE
Feature Request: Option to Set Elementor as Default

### DISCUSSION THREAD

**Discussion by sflwa** [elementor/elementor#28818] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

When using Elementor hosting - Elementor is the default editor and the edit links in WordPress are modified - I'm asking for this to be available on non hosted sites

### Describe the solution you'd like.

Under Features there would be an option to make Elementor the default editor

### Describe alternatives you've considered.

There is a plugin - https://github.com/heyfletch/make-elementor-default-editor that does this but it has not be updated in 5 years - it would be great for this to be in core

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by hitsgee** [comment]
I would also love to have this functionality. It would also be great to have the option what post type it applies to such as pages, posts, products or cpt's

**Comment by steppohub** [comment]
This is a critical feature. We now use Elementor for 90% of our sites and users are constantly getting tricked by inadvertently going back to the Block editor. The plugin: https://github.com/heyfletch/make-elementor-default-editor works ok but not very well and is not maintained or available on the Wordpress repository. We urgently need:

- A change to Elementor so that when you hit 'Exit page' (it should NOT be called 'Exit to Wordpress') it should take you to All Posts or the WP Dashboard by default, not the block editor. If you are required by Wordpress to have an 'exit to Wordpress' then make that another option, not the main one. 
- An option to use Elementor as the default editor (in a good way) - for all pages or ideally for pages/posts/CPTs etc as per @hitsgee.
- A checkbox to lock down the whole site so it ONLY uses Elementor.

Elementor is a lovely system that's growing on me, but you're shooting yourselves in the foot with this constant flipping between Elementor and 'Wordpress' (block editor).

Thanks.
